servers/README.md

```diff
@@ -0,0 +1,32 @@
+# Tonomy Servers
+
+## SSH to server
+
+```bash
+# replace development with master | testnet | development
+source ./server.sh development
+ssh_to_server
+```
+
+## Setup server
+
+```bash
+# replace development with master | testnet | development
+source ./server.sh development
+local_copy_files_to_server
+ssh_to_server
+```
+
+Then run all the commands found in the functions `server.sh`
+
+```bash
+server_setup_prerequisits
+server_setup_tonomy_first_time
+server_setup_ssl
+```
+
+Next time, to reset and re-initalize
+
+```bash
+server_reset
```

Comment:
here when we call the server.sh function it display all commands for that functions and then we can run those manullay?

Reply:
yes correct. need to ssh into the machine and _then_ copy the commands to setup/run the service